client/jetpack-cloud/sections/manage/pricing/pricing-license-selector/style.scss

```diff
@@ -11,23 +10,25 @@
 	.pricing-license-selector {
 		display: flex;
 		align-items: center;
+		justify-content: space-evenly;
 		background: #f4f4f4;
 		border-radius: 8px; /* stylelint-disable-line scales/radii */
 		width: 682px;
-		height: 30px;
 		margin: 20px auto;
-		padding: 1px 2px;
+		padding: 2px 2px;
 		flex-wrap: nowrap;
 		overflow: auto;
 	}
 
 	.pricing-license-option-outer {
 		justify-content: center; /* Centers the inner div horizontally */
-		align-items: center; /* Centers the inner div vertically */
 		display: flex;
 		flex: 0 0 auto;
 		width: 113px;
-		height: 30px;
+		height: 100%;
+		min-height: 30px;
+		// margin: 5px 0;
```

Comment:
We can remove these comments if they are no longer needed.

Reply:
Good catch! Removed in 26e6c7d5f4.